src/content/docs/infrastructure/install-infrastructure-agent/configuration/configure-infrastructure-agent.mdx

```diff
@@ -144,19 +135,27 @@ Here are detailed descriptions of each configuration method:
   </Collapser>
 </CollapserGroup>
 
-<Callout variant="important">
-  For a better understanding of configuration settings, see [Infrastructure configuration settings](/docs/infrastructure/install-configure-infrastructure/configuration/infrastructure-configuration-settings), which contains a detailed explanation of each variable.
-</Callout>
+## Explore configuration options
+
+For a list of all settings you can configure, including definitions, defaults, and related environment variables, see [Infrastructure configuration settings](/docs/infrastructure/install-configure-infrastructure/configuration/infrastructure-configuration-settings).
 
 ## Configuration file structure [#config_structure]
 
-When editing `newrelic-infra.yml`, you must conform to the [YAML format](https://en.wikipedia.org/wiki/YAML) and adhere to the following rules:
+When editing `newrelic-infra.yml`:
 
+* Follow [YAML formatting conventions](https://en.wikipedia.org/wiki/YAML).
```

Comment:
Do we want to have a wikipedia link in our docs?

Reply:
Yeah I wondered about that when I was editing. It's a funny one isn't it? We link to Wikipedia in a few places and I feel comfortable with it, at least when we don't have a better place to send folks.